jsconfig.json

```diff
@@ -2,20 +2,48 @@
   "compilerOptions": {
     "baseUrl": ".",
     "paths": {
-      "*": ["src/*"],
-      "actions/*": ["src/actions/*"],
-      "api/*": ["src/api/*"],
-      "components/*": ["src/components/*"],
-      "constants/*": ["src/constants/*"],
-      "containers/*": ["src/containers/*"],
-      "hooks/*": ["src/hooks/*"],
-      "locale/*": ["src/locale/*"],
-      "reducers/*": ["src/reducers/*"],
-      "screens/*": ["src/screens/*"],
-      "selectors/*": ["src/selectors/*"],
-      "store/*": ["src/store/*"],
-      "utils/*": ["src/utils/*"],
-      "navigators/*": ["src/navigators/*"]
+      "*": [
```

Comment:
What rule is forcing this change? I like the older version better.

Reply:
Just ran prettier auto fix, not sure